ansible/README.md

```diff
@@ -44,6 +44,7 @@ Notes:
 
 ## Roles
 - [ceph-collectd](./roles/ceph-collectd/): Used for ceph cluster hosts
+- [ceph-graphite](./roles/ceph-graphite/): Used for the dashboard host
```

Comment:
Since we do not care about backwards compatibility, can we drop graphite and collectd altogether? What do you guys think?

Reply:
btw: What is our upgrade plan? I suppose we will recommend to purge and re-create the cluster, right?

In that case, we should not really remove these bits from the `purge.yml`.

Reply:
I believe we still need to maintain support for the existing collectors.

There's no real upgrade plan, though; users wanting to go with prometheus will have to start fresh. Purging should allow reuse of an existing dashboard host, though.